README.md

```diff
@@ -1,16 +1,20 @@
-# Title
+# Introduction to Building Complex Class Objects in Ruby
 
-## Learning Goals
+As we covered previously, we use classes as blueprints for objects. Objects are
+the basic building-blocks of Ruby code (everything in Ruby is an object). When
+defining a class, we typically focus on two main properties: states and
+behaviors. Ruby classes are the blueprints that establish what attributes (also
+known as states) and behaviors (known in Ruby as methods) that an object should
+have.
```

Comment:
This brick is mainly about taking the concepts covered in intro to OO and combining them all into fully developed classes. Maybe talk about how we have introduced instance methods and variables and attribute accessors independently of each other, and now we're going to start using them in combination.